lib/neo4j-core/query_clauses.rb

```diff
@@ -398,22 +402,23 @@ def from_key_and_value(key, value)
 
         class << self
           def clause_string(clauses)
-            clauses.map!(&:value).join(', ')
+            clauses.map!(&:value).join(Clause::COMMA_SPACE)
           end
         end
       end
 
       class LimitClause < Clause
         @keyword = 'LIMIT'
+        @keyword_downcase = keyword.downcase
```

Comment:
You might want to consider using a method to get `keyword_downcase` and memoizing it to DRY this up a bit